tests/test_tags.py

```diff
@@ -429,6 +434,24 @@ def test_linux_platforms_32bit_on_64bit_os(self, is_64bit_os, is_x86, monkeypatc
         linux_platform = list(tags._linux_platforms(is_32bit=True))[-1]
         assert linux_platform == "linux_i686"
 
+    def test_linux_platforms_64bit_on_64bit_os_arm(
```

Comment:
Could you make this a parameterized test?
```suggestion
    @pytest.mark.parametrize("is_32bit,expect", [(False, "linux_aarch64"), (True, "linux_armv71")])
    def test_linux_platforms_64bit_on_64bit_os_arm(
```

Reply:
This actually mimics the Intel CPU version. Perhaps we can do another PR that improves the tests for both types of CPUs?

Reply:
If there's another test to touch up then feel free to do it here (it's obviously best practice to do a separate PR, I would rather fix both instead of commit a test that could be structured better).